src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Handlers/Building/ChangeBuildingMeasurementLambdaHandler.cs

```diff
@@ -0,0 +1,66 @@
+namespace BuildingRegistry.Api.BackOffice.Handlers.Lambda.Handlers.Building
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Abstractions.Validation;
+    using Be.Vlaanderen.Basisregisters.AggregateSource;
+    using Be.Vlaanderen.Basisregisters.Sqs.Exceptions;
+    using Be.Vlaanderen.Basisregisters.Sqs.Lambda.Handlers;
+    using Be.Vlaanderen.Basisregisters.Sqs.Lambda.Infrastructure;
+    using Be.Vlaanderen.Basisregisters.Sqs.Responses;
+    using BuildingRegistry.Api.BackOffice.Abstractions.Building.Validators;
+    using BuildingRegistry.Building;
+    using BuildingRegistry.Building.Exceptions;
+    using Microsoft.Extensions.Configuration;
+    using Requests.Building;
+    using TicketingService.Abstractions;
+
+    public sealed class ChangeBuildingMeasurementLambdaHandler : BuildingLambdaHandler<ChangeBuildingMeasurementLambdaRequest>
+    {
+        public ChangeBuildingMeasurementLambdaHandler(
+            IConfiguration configuration,
+            ICustomRetryPolicy retryPolicy,
+            ITicketing ticketing,
+            IIdempotentCommandHandler idempotentCommandHandler,
+            IBuildings buildings)
+            : base(
+                configuration,
+                retryPolicy,
+                ticketing,
+                idempotentCommandHandler,
+                buildings)
+        { }
+
+        protected override async Task<ETagResponse> InnerHandle(ChangeBuildingMeasurementLambdaRequest request, CancellationToken cancellationToken)
+        {
+            var cmd = request.ToCommand();
+
+            try
+            {
+                await IdempotentCommandHandler.Dispatch(
+                    cmd.CreateCommandId(),
+                    cmd,
+                    request.Metadata,
+                    cancellationToken);
+            }
+            catch (IdempotencyException)
+            {
+                // Idempotent: Do Nothing return last etag
+            }
+
+            var lastHash = await GetHash(new BuildingPersistentLocalId(request.BuildingPersistentLocalId), cancellationToken);
+            return new ETagResponse(string.Format(DetailUrlFormat, request.BuildingPersistentLocalId), lastHash);
+        }
+
+        protected override TicketError? InnerMapDomainException(DomainException exception, ChangeBuildingMeasurementLambdaRequest request)
+        {
+            return exception switch
+            {
+                BuildingHasInvalidStatusException => ValidationErrors.ChangeBuildingMeasurement.BuildingInvalidStatus.ToTicketError(),
+                BuildingHasInvalidGeometryMethodException => ValidationErrors.ChangeBuildingMeasurement.BuildingIsOutlined.ToTicketError(),
+                BuildingHasBuildingUnitsOutsideBuildingGeometryException => ValidationErrors.ChangeBuildingMeasurement.BuildingHasBuildingUnitsOutsideChangedGeometry.ToTicketError(),
```

Comment:
I don' think the BuildingHasBuildingUnitsOutsideBuildingGeometryException will be thrown? In case the building unit geometry is outside the new building geometry, the building unit geometry will be derived.